migrations/manual/educandu-2021-12-10-01-migrate-image-tiles-to-use-docs.js

```diff
@@ -0,0 +1,74 @@
+async function updateAll(collection, query, updateFn) {
+  const cursor = collection.find(query);
+
+  /* eslint-disable-next-line no-await-in-loop */
+  while (await cursor.hasNext()) {
+    /* eslint-disable-next-line no-await-in-loop */
+    const doc = await cursor.next();
+    /* eslint-disable-next-line no-await-in-loop */
+    await updateFn(doc);
```

Comment:
Actually I think we could save a lot of time if we would update only documents that actually have changed. We could achieve that by returning it from `updateFunc`. So in case it returns `undefined` or `null` it would not call `replaceOne`. What do you think?

Reply:
Okay, done